apps/web/src/icons/CogIcon.tsx

```diff
@@ -1,22 +1,15 @@
 export default function CogIcon() {
   return (
-    <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <g clipPath="url(#clip0_997_14860)">
-        <path
-          d="M10.1335 12.7334L9.4668 14.6667H6.53346L5.8668 12.7334L5.00013 12.2L3.00013 12.6L1.4668 10.0667L2.8668 8.53337V7.46671L1.4668 5.93337L3.00013 3.40004L5.00013 3.80004L5.8668 3.26671L6.53346 1.33337H9.4668L10.1335 3.26671L11.0001 3.80004L13.0001 3.40004L14.5335 5.93337L13.1335 7.46671V8.53337L14.5335 10.0667L13.0001 12.6L11.0001 12.2L10.1335 12.7334Z"
-          stroke="black"
-        />
-        <path
-          d="M8 10C9.10457 10 10 9.10457 10 8C10 6.89543 9.10457 6 8 6C6.89543 6 6 6.89543 6 8C6 9.10457 6.89543 10 8 10Z"
-          stroke="black"
-          strokeMiterlimit="10"
-        />
-      </g>
-      <defs>
-        <clipPath id="clip0_997_14860">
-          <rect width="16" height="16" fill="white" />
-        </clipPath>
-      </defs>
+    <svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
the old cog icon was too small. I confirmed we aren't using it anywhere else, so there should be no regressions